### ISSUE TITLE
Use prebuilt Mobilizon release

### ISSUE THREAD

**Issue by oncletom** [YunoHost-Apps/mobilizon_ynh#75]
### Describe the bug

Installing Mobilizon requires a bunch of dependencies (system softwares, Node.js, Elixir) just to _build_ the app. It does not go well on 2GB VMs having a NextCloud running for example.

The installer exits when webpack tries to produce a "modern" build.

### Solution

[Use the prebuilt release](https://framagit.org/framasoft/mobilizon/-/merge_requests/774) ~~once it is ready~~ as it is ready. 

In term of impact:

- imagemagick and other image libs won't be needed
- node.js won't be needed
- elixir won't be needed — a standalone runtime will be bundled
- configuration will change, and rely solely on environment variables ([the ones listed in `releases.exs`](https://framagit.org/framasoft/mobilizon/-/blob/master/docker/production/releases.exs))

Maybe other things I missed?

**Comment by yalh76** [comment]
Great, What are the link to download the prebuilt files, I don't understand  where I can find them.
There are release for X86_64, arm, arm64 ?

**Comment by oncletom** [comment]
They are not yet available :) The PR is ongoing on the Mobilizon side.

As far as I understand the [Elixir `mix release` command](https://hexdocs.pm/mix/Mix.Tasks.Release.html), it runs on the same architecture target as the building OS. If there is an `arm64` based runner, there are ways to target the same CI build with a different environment (and flag the downloadable artifacts accordingly).

Does it make sense to you?

**Comment by arthurlogilab** [comment]
I second this proposition, my recent attempts to upgrade from 1.0.0 to 1.0.3 seem to fail (timeout first, now site unavailable and VM has a bunch of beam.smp  processes eating up CPU from `mix compile`)... @yalh76 do you want me to file a separate issue ?

**Comment by arthurlogilab** [comment]
(after a bit of time compiling, the instance came back online, and status in yunohost is correct)

**Comment by yalh76** [comment]
> 
> 
> They are not yet available :) The PR is ongoing on the Mobilizon side.
> 
> As far as I understand the [Elixir `mix release` command](https://hexdocs.pm/mix/Mix.Tasks.Release.html), it runs on the same architecture target as the building OS. If there is an `arm64` based runner, there are ways to target the same CI build with a different environment (and flag the downloadable artifacts accordingly).
> 
> Does it make sense to you?

Yes it makes a lot of sense, it's what is done for pleroma, I retrieve the artefacts from the builds...

**Comment by oncletom** [comment]
A first (should-be) working version produced with Framagit CI is available here: https://framagit.org/oncletom/mobilizon/-/package_files/69/download

URLs should be more predictable than that in the future.
But it's a good start to install Mobilizon without having to install Elixir, nor Node.js.

**Comment by yalh76** [comment]
so cool, I will add that to my todo list ^^
